docs/how-to-open-a-pull-request.md

```diff
@@ -65,7 +65,9 @@ Some examples of good PRs titles would be:
 
 5. Indicate if you have tested on a local copy of the site or not.
 
-   This is very important when making changes that are not just edits to text content like documentation or a challenge description. Examples of changes that need local testing include JavaScript, CSS, or HTML which could change the functionality or layout of a page.
+   - This is very important when making changes that are not just edits to text content like documentation or a challenge description. Examples of changes that need local testing include JavaScript, CSS, or HTML which could change the functionality or layout of a page.
+
+   - These changes should be accompanied by corresponding [Cypress integration tests](/how-to-add-cypress-tests).
```

Comment:
```suggestion
   - If your PR affects the behaviour of a page it should be accompanied by corresponding [Cypress integration tests](/how-to-add-cypress-tests).
```
We don't (yet) test for changes in style, so it's probably better to leave this a little more relaxed.